frontend/public/components/namespace.jsx

```diff
@@ -138,6 +146,9 @@ const NamespacesTableRow = ({ obj: ns, index, key, style }) => {
         <LabelList kind="Namespace" labels={ns.metadata.labels} />
       </TableData>
       <TableData className={namespacesColumnClasses[3]}>
+        <Timestamp timestamp={ns.metadata.creationTimestamp} />
```

Comment:
Most other tables use `fromNow` directly instead of `Timestamp`.

https://github.com/openshift/console/blob/master/frontend/public/components/default-resource.jsx#L80

I think we should be consistent.

Reply:
Fixed.